src/game/core/explore.js

```diff
@@ -16,16 +16,12 @@ import { Observable } from 'rx'
 
 import models from '../models'
 import { run } from './combat'
+import { weightedPool, pickRandom } from './utils'
```

Comment:
Yeah, I wasn't sure where to place this small/general use functions.
I'll move them